core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -749,7 +772,7 @@ public long reinitializeFile(LockedInodePath inodePath, long blockSizeBytes, lon
    * @throws FileDoesNotExistException if inode does not exist
    */
   public InodeLockList lockDescendant(LockedInodePath inodePath, LockMode lockMode,
```

Comment:
Can this be `private`, or just incorporated into `lockDescendantPath`?

Reply:
made `private`